publish-cli/src/test/java/com/ericsson/eiffel/remrem/publish/cli/CliUnitTests.java

```diff
@@ -112,44 +101,45 @@ public void testRunCliOptionsMissingFlags() throws Exception {
 
     @Test
     public void testCreateExchangeDisable() throws Exception {
-        PowerMockito.mockStatic(PublishUtils.class);
-        Mockito.when(PublishUtils.getMessageService(Mockito.eq("eiffelsemantics"), Mockito.any()))
-                .thenReturn(eiffelsemanticsMsgService);
-        Mockito.when(eiffelsemanticsMsgService.getServiceName()).thenReturn("eiffelsemantics");
-        Mockito.doThrow(new RemRemPublishException("message")).when(rmqHelper)
-                .rabbitMqPropertiesInit(Mockito.anyString());
-        File file = new File("src/test/resources/publishMessages.json");
-        if (file.exists()) {
-            System.out.println("fileExist");
+        try (MockedStatic<PublishUtils> utils = Mockito.mockStatic(PublishUtils.class)) {
```

Comment:
Repetitive code better to initialize this before each test using @Before annotation and used elsewhere

Reply:
Moved to setUp().